core/src/main/java/nu/yona/server/properties/YonaProperties.java

```diff
@@ -13,10 +13,10 @@
 
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.boot.context.properties.NestedConfigurationProperty;
-import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
 
 @ConfigurationProperties("yona")
-@Configuration
+@Component
```

Comment:
Why is this no configuration any more?

Reply:
Look at [this Stackoverflow posting](https://stackoverflow.com/questions/39174669/what-is-the-difference-between-configuration-and-component-in-spring/39175018#39175018). I believe marking it with ``@Configuration`` was a mistake in the past.